src/app-layer-protos.c

```diff
@@ -62,6 +62,7 @@ const AppProtoStringTuple AppProtoStrings[ALPROTO_MAX] = {
     { ALPROTO_TELNET, "telnet" },
     { ALPROTO_WEBSOCKET, "websocket" },
     { ALPROTO_LDAP, "ldap" },
+    { ALPROTO_DOH2, "doh2" },
```

Comment:
is this an generally used name for DNS over HTTP/2? Or are ppl just using doh? could perhaps use `dns-http2` or similar